gulpfile.js

```diff
@@ -76,6 +79,19 @@ gulp.task("copy-uswds-js", () => {
   return gulp.src(`${uswds}/js/**/**`).pipe(gulp.dest(`${JS_DEST}`));
 });
 
+gulp.task('build-js', function() {
+  return gulp.src(JS_FILES)
+    .pipe(sourcemaps.init())
+    // Minify the file
+    .pipe(uglify())
+    .pipe(rename({
+      suffix: '.min'
+    }))
+    .pipe(sourcemaps.write('.'))
+    // Output
```

Comment:
use double quotes throughout (we're copying the style from uswds-gulp, it's not based on any other intentional decision making on our end)